src/test/java/milk/manager/iot/WriterTest.java

```diff
@@ -0,0 +1,90 @@
+package milk.manager.iot;
+
+import milk.products.iot.Cheese;
+import milk.products.iot.Milk;
+import milk.products.iot.MilkProduct;
+import milk.products.iot.SourCream;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
You should avoid using ```.*``` import syntax.

```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```
since you are not using other assertions in this file